dist/orejime.css

```diff
@@ -0,0 +1,349 @@
+/*! orejime v1.2.3 - BSD-3-Clause license, original work Copyright (c) 2018 DPKit, modified work Copyright (c) 2019 Empreinte Digitale, all rights reserved. */
```

Comment:
remove dist css and replace with src scss, also will you not add a unique scss file for dark mode?

Reply:
Maybe we should also rename orejime to orejime-light

Reply:
Yes might be better... I'll do that make 2 different scss files